pkg/koordlet/metricsadvisor/performance_collector_linux.go

```diff
@@ -175,7 +178,12 @@ func (c *performanceCollector) collectContainerPSI() {
 
 func (c *performanceCollector) collectSingleContainerPSI(podParentCgroupDir string, containerStatus *corev1.ContainerStatus, pod *corev1.Pod) {
 	collectTime := time.Now()
-	containerPSI, err := util.GetContainerPSI(podParentCgroupDir, containerStatus)
+	containerPath, err := util.GetContainerCgroupPathWithKube(podParentCgroupDir, containerStatus)
+	if err != nil {
+		klog.Errorf("container path err: %v", err)
```

Comment:
please revise the log message. e.g. `failed to get container path for container %s, err ...`